components/feature/downloads/src/test/java/mozilla/components/feature/downloads/manager/AndroidDownloadManagerTest.kt

```diff
@@ -37,10 +36,12 @@ class AndroidDownloadManagerTest {
 
     @Before
     fun setup() {
+        val currentBytesCopied = ("Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko)" +
+                " Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36").length.toLong()
         download = DownloadState(
             "http://ipv4.download.thinkbroadband.com/5MB.zip",
             "", "application/zip", 5242880,
-            "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36"
+            currentBytesCopied
```

Comment:
Nit: the same as https://github.com/mozilla-mobile/android-components/pull/7698/files#r459114027